src/common/pascalscript/Source/uPSRuntime.pas

```diff
@@ -4227,7 +4227,7 @@ procedure PSDynArraySetLength(var arr: Pointer; aType: TPSTypeRec; NewLength: Lo
     end;
     GetMem(darr, Longint(NewLength * elSize) + SizeOf(TDynArrayRecHeader));
     {$IFDEF CPUX64}
-    darr^.header._Padding:=0;
+    //0xdarr^.header._Padding:=0; //We ignore x64 machines

```

Comment:
have no Idea... Lazarus editor. Anyway it was compilation error there - smth like "no property _Padding for object".